plugins/dynamodb/plugin.spec.yaml

```
@@ -23,21 +23,105 @@ hub_tags:
   keywords: [dynamodb, aws, amazon, database]
   features: []
 enable_cache: true
+
+types:
+  HTTPHeaders:
+    connection:
+      title: Connection
+      type: string
+      description: Connection
+      required: false
+    content-length:
+      title: Content-Length
+      type: string
+      description: Content-length
+      required: false
+    content-type:
+      title: Content-Type
+      type: string
+      description: Content-type
+      required: false
+    date:
+      title: Date
+      type: string
+      description: Date
+      required: false
+    server:
+      title: Server
+      type: string
+      description: Server
+      required: false
+    x-amz-crc32:
+      title: X-Amz-Crc32
+      type: string
+      description: X-amz-crc32
+      required: false
+    x-amzn-requestid:
+      title: X-Amzn-RequestID
+      type: string
+      description: X-amzn-requestID
+      required: false
+  ResponseMetadata:
+    HTTPHeaders:
+      title: HTTP Headers
+      type: HTTPHeaders
+      description: HTTP headers
+      required: false
+    HTTPStatusCode:
+      title: HTTP Status Code
+      type: integer
+      description: HTTPS status code
+      required: false
+    RequestId:
+      title: Request ID
+      type: string
+      description: Request ID
+      required: false
+    RetryAttempts:
+      title: Retry Attempts
+      type: integer
+      description: Retry attempts
+      required: false
+  response:
+    Count:
+      title: Count
+      type: integer
+      description: Count
+      required: false
+    Items:
+      title: Items
+      type: "[]object"
```

```suggestion
      type: "[]object"
```
I'm guessing this can't be typed due to being a database table?

---

Yes :)